DDR4 OC Guide.md

```diff
@@ -479,16 +501,18 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
    * On Intel, tWTRS/L should be left on auto and controlled with tWRRD_dg/sg respectively. Dropping tWRRD_dg by 1 will drop tWTRS by 1. Likewise with tWRRD_sg. Once they're as low as you can go, manually set tWTRS/L.
    * On Intel, changing tCWL will affect tWRRD_dg/sg and thus tWTR_S/L. If you lower tCWL by 1 you need to lower tWRRD_dg/sg by 1 to keep the same tWTR values. Note that this might also affect tWR per the relationship described earlier.
    * <sup>1</sup>Some motherboards don't play nice with odd tCWL. For example, I'm stable at 4000 15-19-19 tCWL 14, yet tCWL 15 doesn't even POST. Another user has had similar experiences. Some motherboards may seem fine but have issues with it at higher frequencies (Asus). Manually setting tCWL equal to tCL if tCL is even or one below if tCL is uneven should alleviate this (eg. if tCL = 18 try tCWL = 18 or 16, if tCL = 17 try tCWL = 16).
-   * The extreme preset is not the minimum floor in this case. tRTP can go as low as 6, while tWTRS/L can go as low as 1 6. Some boards are fine doing tCWL as low as tCL - 6. Keep in mind that this *will* increase the load on your memory controller.
-   * On AMD, tCWL can often be set to tCL - 2 but is known to require higher tWRRD.
+   * The extreme preset is not the minimum floor in this case. tRTP can go as low as 5 (6 with Gear Down Mode on), while tWTRS/L can go as low as 1/6. Some boards are fine doing tCWL as low as tCL - 6. Keep in mind that this *will* increase the load on your memory controller.
+   * On AMD, tCWL can often be set to tCL - 2 but is known to require higher tWRRD. Read to Write burst gap is tCWL-tCL + tRDWR. Since this value normally has a set threshold fr stability, increase the gap between tCWL and tCL will require a higher tRDWR to compensate.
```

Comment:
> Read to Write burst gap is tCWL-tCL + tRDWR. Since this value normally has a set threshold fr stability, increase the gap between tCWL and tCL will require a higher tRDWR to compensate.

Can you clarify?

Reply:
Read to write has a gap, which has a minimum threshold. Let say the ram needs this value at 10 clock cycles, 10nCK aka 5ns at 4000. That 10 burst is attained as tCWL-tCL + tRDWR. the value of that would need to be at or exceed 5ns for the ram to be stable. So if tCWL-tCL (as in the difference between them) increases from 0 to -2, then something else must increase, that something else being tWRRD.

Reply:
I'll remoe this anyways, makes no big dfiff